integration-tests/cucumber/cucumber.spec.js

```diff
@@ -43,472 +44,415 @@ const { DD_HOST_CPU_COUNT } = require('../../packages/dd-trace/src/plugins/util/
 const isOldNode = semver.satisfies(process.version, '<=16')
 const versions = ['7.0.0', isOldNode ? '9' : 'latest']
 
-const moduleType = [
-  {
-    type: 'commonJS',
-    runTestsCommand: './node_modules/.bin/cucumber-js ci-visibility/features/*.feature',
-    runTestsWithCoverageCommand:
-      './node_modules/nyc/bin/nyc.js -r=text-summary ' +
-      'node ./node_modules/.bin/cucumber-js ci-visibility/features/*.feature',
-    parallelModeCommand: './node_modules/.bin/cucumber-js ' +
-    'ci-visibility/features/*.feature --parallel 2',
-    featuresPath: 'ci-visibility/features/',
-    fileExtension: 'js'
-  }
-]
```

Comment:
ℹ️ I slightly refactored this file but GH can't correctly recognize the diff anymore - basically some tests were added for early flake detection for parallel mode